src/app/App.tsx

```diff
@@ -74,103 +68,22 @@ export class App extends React.Component<Props> {
     if (isE2EEnv) {
       LogBox.ignoreAllLogs(true)
     }
-
-    // Handles opening Clevertap deeplinks when app is closed / in background
-    // @ts-expect-error the clevertap ts definition has url as an object, but it
-    // is a string!
-    CleverTap.getInitialUrl(async (err: any, url: string) => {
-      if (err) {
-        if (/CleverTap initialUrl is (nil|null)/gi.test(err)) {
-          Logger.warn('App/componentDidMount', 'CleverTap InitialUrl is nil|null', err)
-        } else {
-          Logger.error('App/componentDidMount', 'App CleverTap Deeplink on Load', err)
-        }
-      } else if (url) {
-        await this.handleOpenInitialURL({ url }, true)
-      }
-    })
-
-    // Handles opening Clevertap deeplinks when app is open
-    CleverTap.addListener('CleverTapPushNotificationClicked', async (event: any) => {
-      // Url location differs for iOS and Android
-      const url = Platform.OS === 'ios' ? event.customExtras['wzrk_dl'] : event['wzrk_dl']
-      if (url) {
-        await this.handleOpenURL({ url }, true)
-      }
-    })
-
-    Linking.addEventListener('url', this.handleOpenURL)
-
-    if (FIREBASE_ENABLED) {
-      this.dynamicLinksRemoveListener = dynamicLinks().onLink(({ url }) =>
-        this.handleOpenURL({ url })
-      )
-
-      const firebaseUrl = await dynamicLinks().getInitialLink()
-      if (firebaseUrl) {
-        await this.handleOpenURL({ url: firebaseUrl.url })
-      }
-    }
-
-    const initialUrl = await Linking.getInitialURL()
-    if (initialUrl) {
-      await this.handleOpenInitialURL({ url: initialUrl })
-    }
-
-    this.logAppLoadTime()
-
-    store.dispatch(appMounted())
-  }
-
-  dynamicLinksRemoveListener: (() => void) | undefined
-
-  logAppLoadTime() {
-    const { appStartedMillis } = this.props
-    const reactLoadDuration = (this.reactLoadTime - appStartedMillis) / 1000
-    const appLoadDuration = (Date.now() - appStartedMillis) / 1000
-    Logger.debug(
-      'App/logAppLoadTime',
-      `reactLoad: ${reactLoadDuration} appLoad: ${appLoadDuration}`
-    )
-    const { width, height } = Dimensions.get('window')
-
-    ValoraAnalytics.startSession(AppEvents.app_launched, {
-      deviceHeight: height,
-      deviceWidth: width,
-      reactLoadDuration,
-      appLoadDuration,
-      language: i18n.language || store.getState().i18n.language,
-    })
   }
 
   componentWillUnmount() {
-    CleverTap.removeListener('CleverTapPushNotificationClicked')
-    Linking.removeEventListener('url', this.handleOpenURL)
-    this.dynamicLinksRemoveListener?.()
     store.dispatch(appUnmounted())
```

Comment:
Maybe we should move it closer to `appMounted`?

Reply:
yes good idea, done!